build/yaml/jobs/linux/tests.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Job Template: Test Linux Binaries
+
+jobs:
+- job: Test_Linux_Binaries
+  displayName: Linux Tests
+
+  pool:
+    ${{ if eq(variables['System.TeamProject'], 'devdiv') }}:
+      name: AzurePipelines-EO
+      demands: ImageOverride -equals AzurePipelinesUbuntu20.04compliant
+    ${{ else }}:
+      vmImage: ubuntu-latest
+
+  workspace:
+    clean: outputs
+
+  variables:
+    configuration: Debug
+
+  steps:
+#  - download: current
+#    patterns: |
+#      binaries-windows-$(Configuration)/AnyCPU/**
+#      binaries-linux-ubuntu-$(Configuration)/ClrInstrumentationEngine/**
```

Comment:
removing; this was the new way to download pipeline artifacts but for some reason the files are inaccessible in subsequent jobs (file/directory not found errors).